src/org/opendatakit/briefcase/operations/PullFormFromCentral.java

```diff
@@ -0,0 +1,133 @@
+import org.opendatakit.briefcase.pull.central.PullFromCentral;
+import org.opendatakit.briefcase.reused.BriefcaseException;
+import org.opendatakit.briefcase.reused.Optionals;
+import org.opendatakit.briefcase.reused.http.CommonsHttp;
+import org.opendatakit.briefcase.reused.http.Credentials;
+import org.opendatakit.briefcase.reused.http.Http;
+import org.opendatakit.briefcase.reused.http.response.Response;
+import org.opendatakit.briefcase.reused.job.JobsRunner;
+import org.opendatakit.briefcase.reused.transfer.CentralServer;
+import org.opendatakit.briefcase.transfer.TransferForms;
+import org.opendatakit.briefcase.util.FormCache;
+import org.opendatakit.common.cli.Args;
+import org.opendatakit.common.cli.Operation;
+import org.opendatakit.common.cli.Param;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PullFormFromCentral {
+  private static final Logger log = LoggerFactory.getLogger(PullFormFromCentral.class);
+  private static final Param<Void> PULL_FROM_CENTRAL = Param.flag("pllc", "pull_central", "Pull form from a Central server");
+
+
+  public static Operation OPERATION = Operation.of(
+      PULL_FROM_CENTRAL,
+      PullFormFromCentral::pullFromCentral,
+      Arrays.asList(STORAGE_DIR, SERVER_URL, PROJECT_ID, CREDENTIALS_EMAIL, CREDENTIALS_PASSWORD),
+      Arrays.asList(FORM_ID, MAX_HTTP_CONNECTIONS)
+  );
+
+  private static void pullFromCentral(Args args) {
+    CliEventsCompanion.attach(log);
+    Path briefcaseDir = Common.getOrCreateBriefcaseDir(args.get(STORAGE_DIR));
+    FormCache formCache = FormCache.from(briefcaseDir);
+    formCache.update();
+    BriefcasePreferences appPreferences = BriefcasePreferences.appScoped();
+    appPreferences.setStorageDir(briefcaseDir);
+
+    int maxHttpConnections = Optionals.race(
+        args.getOptional(MAX_HTTP_CONNECTIONS),
+        appPreferences.getMaxHttpConnections()
+    ).orElse(DEFAULT_HTTP_CONNECTIONS);
+    Http http = appPreferences.getHttpProxy()
+        .map(host -> CommonsHttp.of(maxHttpConnections, host))
+        .orElseGet(() -> CommonsHttp.of(maxHttpConnections));
+
+    CentralServer server = CentralServer.of(args.get(SERVER_URL), args.get(PROJECT_ID), new Credentials(args.get(CREDENTIALS_EMAIL), args.get(CREDENTIALS_PASSWORD)));
+
+    String token = http.execute(server.getSessionTokenRequest())
+        .orElseThrow(() -> new BriefcaseException("Can't authenticate with ODK Central"));
+
+    Response<List<RemoteFormDefinition>> response = http.execute(server.getFormsListRequest(token));
+    if (!response.isSuccess()) {
+      System.err.println(response.isRedirection()
+          ? "Error connecting to Central: Redirection detected"
+          : response.isUnauthorized()
+          ? "Error connecting to Central: Wrong credentials"
+          : response.isNotFound()
+          ? "Error connecting to Central: Central not found"
+          : "Error connecting to Central");
+      return;
```

Comment:
I’m starting to like this form of if/else if/…/else if/else. I prefer that the question mark abut the question, as in English, but I accept your style.